internals/src/macros.rs

```diff
@@ -139,9 +139,12 @@ macro_rules! debug_from_display {
 /// Asserts a boolean expression at compile time.
 #[macro_export]
 macro_rules! const_assert {
-    ($x:expr) => {{
-        const _: [(); 0 - !$x as usize] = [];
-    }};
+    ($x:expr $(; $message:expr)+) => {
```

Comment:
This needs to be `?`, not `+`. If you somehow want to support multiple messages then this syntax would look better: `$(; $(mesage:expr),+ $(,)?)?`

Reply:
Drats, that was just a mistake - I'm sick and brain fried right now, you might have noticed. I'me enjoying coding though. Please excuse me.

Reply:
Used `?`